apps/studio/src/store/index.ts

```diff
@@ -422,6 +422,19 @@ const store = new Vuex.Store<State>({
       }
     },
 
+    async updateTablePartitions(context, table: TableOrView) {
```

Comment:
So we don’t do this with indexes or relations or triggers. You should emulate how those properties tabs work so we’re consistent